client/homebrew/editor/editor.jsx

```diff
@@ -139,10 +139,10 @@ const Editor = createClass({
 
 						// Highlight injectors {style}
 						if(line.includes('{') && line.includes('}')){
-							const regex = /(?<!{){(?=((?::(?:"[\w,\-()#%. ]*"|[\w\-()#%.]*)|[^"':{}\s]*)*))\1}/g;
+							const regex = /(?:^|[^{|\n])({(?=((?::(?:"[\w,\-()#%. ]*"|[\w\-()#%.]*)|[^"':{}\s]*)*))\2})/gm;
```

Comment:
Is there a reason we want to exclude injectors occurring after pipes `|`  with `[^{|\n]` ?

Reply:
No.  When i wrote it I was thinking of it as "OR" but clearly that isn't needed in the brackets. Fixed.

Another question:  Do we want the highlight/injection to happen if there is a `\s` / `space` before the brackets?  

a. `## My text is here {class}`   
b. `## My text is here{class}`

Should both be injected, or just *b*?   

In my opinion, both should work for readability sake.  However, if *A* is allowed then it wouldn't be possible to write "The alien came around the corner {and started blasting}."    Obviously, parentheses would be the logical choice, but it may preclude stylistic choices.

We don't have an escape character for these things, right?

Reply:
For now, I just removed that pipe from the pattern.  So if that works for you, merge it.

Reply:
For now I just pushed the fix to the pipe.  I tested it in Safari.

Reply:
Yeah I'm pretty sure we inject with any number of beginning spaces too.  A backslash will escape the injector

Reply:
okay then i think this is good to go?